icu4c/source/test/testdata/idna_conf.txt

```diff
@@ -919,7 +919,7 @@ type: toascii
 desc: 4-1-2-9 nonbmp<10400><00ED>dn ->\
  nonbmp<10428><00ED>dn
 namebase: nonbmp<10400><00ED>dn
-nameutf8: nonbmp������ídn
+nameutf8: nonbmp������ídn
```

Comment:
The fix I made to this file was on line 1236. Not sure what triggered the above issue, honestly.

```
desc: 5-1-2 Unassinged outside BMP; zone editors should reject
desc: 5-1-2 Unassigned outside BMP; zone editors should reject
```

Reply:
Hmm.. perhaps the encoding of the file got changed when you saved it?
The file is UTF-8 with BOM. When I edit the file with VS Code it makes similar changes, but when I edit with Notepad ++ it does not.

If the only changes you made were on line 1236, then I think these need to be reverted.

Reply:
Hmmm might there have been some change in encoding when saving the file? @jefgen any ideas?

Reply:
It looks like the character "𐐀" (U+10400) is encoded in UTF-8 in the file as a surrogate pair D801 DC00.
I think this is probably fine and/or perhaps intentional for the test case.

When the file is edited/saved it seems that the editor (or program that is changing the file) changes this to use the non-surrogate pair encoding, so it saves it as `F0 90 90 80` instead of `ed a0 81 ed b0 80 c3`.

I don't think we want to change the test case as part of fixing typos though. So I think these changes should be reverted in this file.

Reply:
I'm using gitkraken as my git client. For the life of me the only change I see is line 1236. I so no encoding changes.  
Screenshot:  
![image](https://user-images.githubusercontent.com/4140247/125700075-fd494ef9-3499-4d32-ab48-1b7cf57b5a12.png)

Reply:
Hmm, IIRC, GitKraken is based on Electron, which is also what VS Code uses -- so that might explain why I see the same changes when I edit the file with VSCode.

What does git show in the terminal? When I edit the file with VSCode, `git diff` shows all the extraneous changes.

I wonder, maybe try reverting the changes to the file, and then using a different editor to change the one line, and then stage it using the terminal?